src/background/hypothesis-chrome-extension.js

```diff
@@ -292,12 +310,14 @@ export default function HypothesisChromeExtension({
       return;
     }
 
+    // If user disabled the badge count, this call to `sync.get` will
+    // return `{ badge: false}`
```

Comment:
Is the result after an initial install just `{}`?

Reply:
Yes, `{}`.